pyaerocom/aeroval/compute_fairmode_stats.py

```diff
@@ -0,0 +1,98 @@
+"""
+Functions for computing the FAIRMODE statistics
+"""
+
+from math import sqrt, isclose
+from typing import Tuple
+
+
+def _get_RMSU(mean: float, std: float, species: str) -> Tuple[float, float, float, float]:
+    species_values = dict(
+        concno2=dict(
+            UrRV=0.24,
+            RV=200,
+            alpha=0.2,
+        ),
+        vmro3=dict(
+            UrRV=0.18,
+            RV=120,
+            alpha=0.79,
+        ),
+        concpm10=dict(
+            UrRV=0.28,
+            RV=50,
+            alpha=0.13,
+        ),
+        concpm25=dict(
+            UrRV=0.28,
+            RV=25,
+            alpha=0.3,
+        ),
+    )
+
+    if species not in species_values.keys():
+        raise KeyError(f"Species {species} not in list {species_values.keys()}")
```

Comment:
You do not need `.keys()` to check if `species` is a key on the dict.
Also, from the point of view of the user of this function this is a `ValueError`.
I mean, the function was called with an unsupported value...

``` python
    if species not in species_values:
        raise ValueError(f"Unsupported {species=})
```

Reply:
Addressed